.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -28,8 +28,10 @@ Please try to include the relevant sections in your issue description.
 You can upload the whole `crash.log` file (zipped if necessary) on GitHub by dragging and dropping it onto this textbox.
 
 If your issue doesn't directly concern a Lua crash, we'll quite likely need you to reproduce the issue with *verbose* debug logging enabled before providing the logs to us.
-To do so, from the file manager, go to [Tools] → More tools → Developer options, and tick both `Enable debug logging` and `Enable verbose debug logging`.
-You'll need to restart KOReader after toggling these on.
+To do so, go to `Top menu - Hamburger menu - Help - Report a bug` and tap `Enable verbose logging`. Restart as requested, then repeat the steps for your issue.
+
+Alternatively, from the file manager, go to [Tools] → More tools → Developer options, and tick both `Enable debug logging` and `Enable verbose debug logging`.
```

Comment:
Instructions above are simple, elegant. Why alternatives?

Reply:
For legacy purposes? :D

Reply:
I share your desire to add too much information, but it'll be clearer with just one. If someone props up with a version from two years ago we'll tell them to try the most recent version first regardless.